universal-application-tool-0.0.1/app/models/Application.java

```diff
@@ -16,18 +16,22 @@
 
   @ManyToOne private Program program;
 
+  @Constraints.Required private LifecycleStage lifecycleStage;
+
+  @UpdatedTimestamp private Instant submitTime;
+
   // used by generated code
   @SuppressWarnings("UnusedVariable")
   @Constraints.Required
   @DbJson
   private String object;
```

Comment:
Question (non-blocking): Is the plan to have separate applications and a consolidated profile (bank of answers) for an applicant? Feel free to point me to any doc that I missed.